pkg/utils/watchChaosContainer.go

```diff
@@ -2,32 +2,57 @@ package utils
 
 import (
 	"time"
-
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+
+var (
+	//chaosPod holds the experiment pod spec
+	chaosPod = &corev1.Pod{}
```

Comment:
Are we using this global definition somewhere? better approach is to define this inside the method itself

Reply:
Done!